lib-multisrc/mangabox/src/eu/kanade/tachiyomi/multisrc/mangabox/MangaBox.kt

```diff
@@ -346,11 +346,11 @@ abstract class MangaBox(
     }
 
     override fun pageListParse(document: Document): List<Page> {
-        val element = document.select("head > script").lastOrNull()
+        val content = document.selectFirst("script:containsData(cdns =)")?.data()
```

Comment:
Seems brittle. Could you do something like this? Needs testing:
```suggestion
        val content = document.select("script:containsData(cdns =)").joinToString("\n") { it.data() }
```

Mostly to project against them moving script tags around.